components/Footer.vue

```diff
@@ -1,10 +1,44 @@
 <template>
-    <div class="flex bg-teal-700 text-white">
-        <div class="flex-auto">
-            <a href="https://www.netlify.com/" target="_blank">Deployed on Netlify</a>
+    <div class="footer flex bg-orange-100 text-sm text-current font-semibold py-2">
+        <div class="left-footer-section flex flex-0 justify-start">
+            <NuxtLink to="/" class="self-center ml-8 mr-2">
+                © 2023 Find a doc JP
+            </NuxtLink>
+            <span class="self-center">·</span>
+            <nav class="flex mx-2">
+                <NuxtLink to="/privacypolicy" class="self-center">
+                    Privacy
+                </NuxtLink>
+            </nav>
+            <span class="self-center">·</span>
+            <nav class="flex mx-2">
+                <NuxtLink to="/terms" class="self-center">
+                    Terms
+                </NuxtLink>
+            </nav>
         </div>
-        <div class="flex-auto">
-            Contribute on <a href="https://github.com/ourjapanlife" target="_blank">GitHub</a>
+        <div class="middle-footer section flex flex-1 align-middle justify-center">
+            <span class="self-center">Powered by</span>
+            <nav class="flex mx-2 self-center">
+                <NuxtLink to="https://www.netlify.com/" target="_blank" class="underline">
+                    Netlify
+                </NuxtLink>
+            </nav>
+        </div>
+        <div class="right-footer-section flex flex-0 flex-row gap-6 justify-end px-8 text-xs">
+            <div class="flex flex-col flex-wrap mx-2">
```

Comment:
added this in. I think it would be valuable to have a link to our github.  We can change the design however, just wanted to get it in the code